index.html

```diff
@@ -22,7 +22,7 @@ <h1>&#128075; Welcome!</h1>
 <section class="pure-g" id="join-the-community">
   <div class="pure-u-1">
     <h2>Join the Community</h2>
-    <p>For code chat, job listings, ruby help, keyboard nerdery, and banter, join us on <a href="https://slack.ruby.nz">Ruby Oceania Slack</a>.</p>
+    <p>For code chat, job listings, ruby help, keyboard nerdery, and banter, join us on <a href="mailto:secretary@ruby.nz?subject=Can I have Ruby Oceania Slack access please?">Ruby Oceania Slack</a>.</p>
```

Comment:
we have a link to `slack.ruby.nz` just a few lines down

Reply:
@parndt got it now.